services/brig/src/Brig/User/Email.hs

```diff
@@ -152,6 +149,38 @@ renderDeletionEmail DeletionEmailTemplate{..} DeletionEmail{..} =
     replace2 x      = x
 
 -------------------------------------------------------------------------------
+-- Verification Email
+
+data VerificationEmail = VerificationEmail
+    { vfTo   :: !Email
+    , vfPair :: !ActivationPair
+    }
+
+renderVerificationMail :: VerificationEmail -> VerificationEmailTemplate -> Mail
+renderVerificationMail VerificationEmail{..} VerificationEmailTemplate{..} =
+    (emptyMail from)
+        { mailTo      = [ to ]
+        , mailHeaders = [ ("Subject", toStrict subj)
+                        , ("X-Zeta-Purpose", "Verification")
+                        , ("X-Zeta-Key", Ascii.toText key)
```

Comment:
The `key` part is not in the actual email as it isn't used - so I suppose it can be removed from the headers, too?